composer.json

```diff
@@ -38,7 +38,7 @@
     "mheap/phpunit-github-actions-printer": "^1.2",
     "phpstan/phpstan-doctrine": "^0.12.8",
     "phpstan/phpstan-symfony": "^0.12.2",
-    "phpunit/phpunit": "^7.5",
+    "phpunit/phpunit": "^8.5",
```

Comment:
out of the scope but i'd advise to use the simple-phpunit instead (see https://symfony.com/doc/current/components/phpunit_bridge.html)

> The PHPUnit Bridge provides utilities to report legacy tests and usage of deprecated code and helpers for mocking native functions related to time, DNS and class existence.

Reply:
I know it's a recommended way to do it, but we really don't need the simple-phpunit features, and having the phpunit source code in vendors/.bin/phpunit/phpunit-8-5/... with symlinks for children dependencies is something a big ugly and not really well managed by alpine (we can have some symlinks loops)